launcher-scripts/Linux-launcher.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 OLD_JAVA_HOME=$JAVA_HOME
-export JAVA_HOME=`pwd`/jdk8u212-b03-jre
-"$JAVA_HOME/bin/java" -DLAUNCHER_JAVA_PATH="$JAVA_HOME/bin/java" -jar ./libs/admin.jar
-export JAVA_HOME=$OLD_JAVA_HOME
+SCRIPT_DIRECTORY=`(cd $(dirname $0); pwd)`
```

Comment:
At first read I was concerned that we were changing directory etc. So how about we just use the tool `readlink`?

```suggestion
SCRIPT_DIRECTORY=`readlink -f $(dirname $0)`
```